app/src/organisms/ProtocolSetup/RunSetupCard/RobotCalibration/CalibrationItem.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  Text,
+  Flex,
+  Icon,
+  SIZE_2,
+  FONT_SIZE_BODY_1,
+  FONT_SIZE_BODY_2,
+  FONT_STYLE_ITALIC,
+  C_NEAR_WHITE,
+  C_WHITE,
+  COLOR_SUCCESS,
+  SPACING_2,
+  ALIGN_CENTER,
+  DIRECTION_ROW,
+  BORDER_SOLID_MEDIUM,
+  Box,
+  JUSTIFY_SPACE_BETWEEN,
+} from '@opentrons/components'
+import { formatLastModified } from '../../../CalibrationPanels/utils'
+
+interface Props {
+  button?: JSX.Element
+  calibrated: boolean
+  calibratedDate?: string | null
+  index?: number
+  subText?: string
+  title?: string
+}
+
+export function CalibrationItem(props: Props): JSX.Element | null {
+  const { index, title, subText, calibratedDate, calibrated, button } = props
+  const { t } = useTranslation(['protocol_setup'])
```

Comment:
```suggestion
  const { t } = useTranslation('protocol_setup')
```
same reasoning as above